components/dashboard/src/components/CheckBox.tsx

```diff
@@ -30,7 +30,7 @@ function CheckBox(props: {
             {...inputProps}
         />
         <div className="flex flex-col ml-2">
-            <label htmlFor={checkboxId} className="text-gray-800 text-md font-semibold tracking-wide">{props.title}</label>
+            <label htmlFor={checkboxId} className="text-gray-800 dark:text-gray-100 text-md font-semibold tracking-wide">{props.title}</label>
```

Comment:
**issue**: This will need a light background on check, see [relevant specs](https://www.figma.com/file/XQOjjEHO0ALtxdIT9Blzh2/Dark-Theme-3806?node-id=225%3A857).

Reply:
Thanks! Adjusted the checkboxes to the [light](https://www.figma.com/file/tZXINzszBE9rzFvUVzTQa3/Notifications-%233398?node-id=1%3A263) and [dark](https://www.figma.com/file/XQOjjEHO0ALtxdIT9Blzh2/Dark-Theme-3806?node-id=225%3A857) (i.e. with a 2px border again)